aten/src/ATen/native/native_functions.yaml

```diff
@@ -9291,6 +9291,18 @@
 - func: special_erfc.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   python_module: special
 
+- func: special_erfcx(Tensor self) -> Tensor
+  python_module: special
```

Comment:
would be great to have this moved to structured. sorry missed that previously.

You can look at `i0e` PR or just check UnaryOps.cpp to get a feel.

For more inforation on structured kernel: https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md